data/json/recipes/weapon/magazines.json

```diff
@@ -50,6 +50,70 @@
     "tools": [ [ [ "small_repairkit", 10 ], [ "large_repairkit", 5 ] ] ],
     "components": [ [ [ "steel_chunk", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "ammolink223",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_MAGAZINES",
+    "skill_used": "fabrication",
+    "skills_required": [ "gun", 4 ],
```

Comment:
I'm not sure if marksmanship skill has anything to do with forging ammo linkages.

Reply:
Well, it is required for making all the clips/magazines within that file.